frontend/src/pages/ActivityReport/Pages/Review/Approver/Review.js

```diff
@@ -80,24 +82,35 @@ const Review = ({
           </div>
         </Fieldset>
         { !showDraftViewForApproverAndCreator ? (
-          <FormItem
-            name="status"
-            label="Choose report status"
-            className="margin-bottom-3"
-          >
-            <Dropdown
-              id="status"
+          <>
+            {
+            dateSubmitted
+              ? (
+                <Fieldset className="smart-hub--report-legend margin-top-3" legend="Date Submitted">
+                  <Label className="margin-top-1" htmlFor="date_submitted">{formattedDateSubmitted}</Label>
```

Comment:
fieldset/legend are not needed ([they are really only for grouping related controls](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset), I.E. a group of radio buttons or checkboxes)

Reply:
also a label is really just for a form control - I think you just want two paragraph tags here

Reply:
Made both labels.

Reply:
a label is really just for a form control - I think you just want two paragraph tags here